wadas/ui/mainwindow.py

```diff
@@ -472,25 +474,49 @@ def update_en_camera_list(self):
 
     def update_en_actuator_list(self):
         """Method to list enabled actuator(s) in UI"""
-
+        threshold_time = 30
         self.ui.listWidget_en_actuators.clear()
         for actuator in Actuator.actuators.values():
             if actuator.enabled:
-                if actuator.last_update is not None and (
-                    datetime.datetime.now() - actuator.last_update > timedelta(seconds=30)
-                ):
-                    text = f"({actuator.type.value}) {actuator.id} - inactive"
-                    self.ui.listWidget_en_actuators.addItem(text)
-                    item = self.ui.listWidget_en_actuators.item(
-                        self.ui.listWidget_en_actuators.count() - 1
-                    )
-                    item.setForeground(QBrush(QtCore.Qt.GlobalColor.red))
-                    item.setToolTip(
-                        f"Last Activity: {actuator.last_update.strftime('%d %b %Y, %H:%M:%S')}"
-                    )
+                if FastAPIActuatorServer.actuator_server.startup_time:
+                    # inactive actuator: connected at least once but unseen for {threshold_time} seconds
+                    # or never connected within the first 30 seconds since server startup.
+                    if (actuator.last_update is not None and (
+                            datetime.datetime.now() - actuator.last_update > timedelta(seconds=threshold_time)) or
+                            actuator.last_update is None and (
+                                    datetime.datetime.now() - FastAPIActuatorServer.actuator_server.startup_time > timedelta(
+                                seconds=threshold_time))):
+
+                        text = f"({actuator.type.value}) {actuator.id} - inactive"
+                        color = QtCore.Qt.GlobalColor.red
```

Comment:
I would prefer to keep black color for inactive (yet not connected) actuators. Using red for timeout and green for connected ones

Reply:
I used the term "inactive" to indicate the "timed out actuators"